### PR TITLE
feat: usdt v3 ethereum

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: script/DeployUSDTV3Utils.s.sol [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import {WithChainIdValidation} from './WithChainIdValidation.sol';
+import {Script} from 'forge-std/Script.sol';
+import {AaveV3Ethereum} from 'aave-address-book/AaveV3Ethereum.sol';
+import {WadRayMath} from 'aave-v3-core/contracts/protocol/libraries/math/WadRayMath.sol';
+import {DefaultReserveInterestRateStrategy} from 'aave-v3-core/contracts/protocol/pool/DefaultReserveInterestRateStrategy.sol';
+
+contract DeployMainnetPayload is WithChainIdValidation {
+  constructor() WithChainIdValidation(1) {}
+
+  // TODO add function to deploy payload
```
Comment (rustboyar) [inline script/DeployUSDTV3Utils.s.sol:12]: i think this TODO is not needed anymore

Comment (rustboyar) [inline script/DeployUSDTV3Utils.s.sol:12]: i think this TODO is not needed anymore

File: src/test/mainnet/AaveV3USDTETHPayloadTest.t.sol [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import {AaveV3EthUSDTPayload} from '../../contracts/mainnet/AaveV3EthUSDTPayload.sol';
+
+contract AaveV3EthUSDTPayloadTest is ProtocolV3_0_1TestBase, TestWithExecutor {
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 16576447);
+    _selectPayloadExecutor(AaveGovernanceV2.SHORT_EXECUTOR);
+  }
+
+  function testPoolActivation() public {
+    AaveV3EthUSDTPayload USDT = new AaveV3EthUSDTPayload();
```
Comment (sendra) [inline src/test/mainnet/AaveV3USDTETHPayloadTest.t.sol:17]: This provably would be better as USDTPayload. less confusing

File: script/DeployUSDTV3Utils.s.sol (hunk 1 above)
Comment (sendra) [reply script/DeployUSDTV3Utils.s.sol:12]: Payload deployment code missing

**Review by sendra — APPROVED** [review]

**Pull request merged**